db/index.py

```diff
@@ -189,6 +189,19 @@ def get_push_key():
     # [END push_key]
     print(post_id)
 
+def run_transaction():
+    # [START transaction]
+    def increment_votes(current_value):
+        return current_value + 1 if current_value else 1
+
+    upvotes_ref = db.reference('server/saving-data/fireblog/posts/-JRHTHaIs-jNPLXOQivY/upvotes')
```

Comment:
I'd probably make run_transaction take `upvotes_ref` as a parameter rather than having this strange hard-coded path